CMakeLists.txt

```diff
@@ -509,6 +509,12 @@ if (BUILD_HIPIFY_CLANG)
     add_dependencies(pkg_hip_base hipify-clang)
 endif()
 
+if(HIP_PLATFORM STREQUAL "vdi")
+    add_custom_target(hip_bins
+      COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/bin/hipcc -o ${PROJECT_BINARY_DIR}/hipInfo -v ${CMAKE_CURRENT_SOURCE_DIR}/samples/1_Utils/hipInfo/hipInfo.cpp
+      COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/bin/hipcc -o ${PROJECT_BINARY_DIR}/hipBusBandWidth ${CMAKE_CURRENT_SOURCE_DIR}/samples/1_Utils/hipBusBandwidth/hipBusBandwidth.cpp ${CMAKE_CURRENT_SOURCE_DIR}/samples/1_Utils/hipBusBandwidth/ResultDatabase.cpp)
```

Comment:
These should be built with the `CMAKE_CXX_COMPILER`. Using hipcc will be broken if the compiler is installed into /usr or /usr/local, even though the user has set the `CMAKE_CXX_COMPILER` to the clang++ that supports hip.